applications/frontier-pos-template.md

```diff
@@ -0,0 +1,184 @@
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. See the [delivery guidelines](https://grants.web3.foundation/docs/Support%20Docs/milestone-deliverables-guidelines#testing-guide) for details. |
+| **0d.** | Docker                    | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                                                                                                  |
+|     0e. | Article                   | We will publish an **article** that explains [...] (what was done/achieved as part of the grant). (Content, language, and medium should reflect your target audience described above.)                                                                                                                         |
+|      1. | Client                    | An NPOS consensus and EVM-compatible client.                                                                                                                                                                                                                                                                                             |
+|      2. | runtime                   | An NPOS consensus and EVM-compatible runtime.                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                        |
+|      3. | Substrate chain           | An EVM-compatible chain with NPOS consensus.                                                                                                                                                                                                                                                                                               |
+|     4.  | Friendly code structure                     | A developer-friendly code structure, similar to the one found in https://github.com/substrate-developer-hub/substrate-node-template.git.                                                                                                                                                                                                                        |
```

Comment:
This does not sound like a deliverable, but a feature of a deliverable. I suggest to remove it or make it part of one of the other deliverables.

Reply:
> This does not sound like a deliverable, but a feature of a deliverable. I suggest to remove it or make it part of one of the other deliverables.

The deliverable is a fully functional solo chain with NPOS and EVM compatibility.